yogstation/code/game/gamemodes/battle_royale/battleroyale.dm

```diff
@@ -225,7 +225,7 @@ GLOBAL_VAR(stormdamage)
 		qdel(I)
 	tfue.equipOutfit(/datum/outfit/battleroyale, visualsOnly = FALSE)
 
-/mob/living/carbon/human/Life()
+/mob/living/carbon/human/Life(seconds_per_tick = SSMOBS_DT, times_fired)
 	. = ..()
 	if(is_battleroyale(src))
 		var/datum/antagonist/battleroyale/gamer = mind.has_antag_datum(/datum/antagonist/battleroyale)
```

Comment:
This should probably all be moved to ``/mob/living/carbon/human/Life()``, it is easy to break here.

Even better alternative, do what Bloodsuckers do and register signal to comsig_mob_life.